### ISSUE TITLE
Book 05: Sentence 070

### ISSUE THREAD

**Issue by ftyers** [ftyers/UD_Classical_Nahuatl-FloCo#37]
![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/83283df0-55d0-4606-b128-f816996f3245)

```
# sent_id = Book_05_-_The_Omens.txt:70
# text = No ipan qujmatia, ca ixiptla in tlacateculotl, tezcatlipuca: auh in jquac mjexi, in teiexi, qujtoaia: omjex in tezcatlipuca.
# text[norm] = No ipan quimatia, ca ixiptla in tlacatecolotl, Tezcatlipoca: auh in ihcuac miyexi, in teiyexi, quihtoaya: omiyex in Tezcatlipoca.
# text[orig] = No·ipan¶qujmatia·,·ca·ixiptla·in·tlacateculotl·,·tezcatlipuca·:·auh·inj·quac·mjexi·,·in¶teiexi·,·qujtoaia·:·omjex·intezca¶tlipuca·.
# text[nhi] = Noyohqui oquinimiliaya, oquixhuicaya in tlacaticolotl, itoca Tezcatlipoca: huan cuando omiguixiya, oquihtoayah: omiguix Tezcatlipoca.
# text[osp] = Tambien dezian, que este animalejo, era ymagen del dioa que llamauan Tezcatlipuca: y quando este animalejo, espelia aquella materia hedionda, que era la orina, o el mismo estiercol, o la ventosidad, dezian: Tezcatlipuca aventosiado.
# text[spa] = También sobre él sabían, que era su imagen del demonio Tezcatlipoca: y cuando echaba un pedo, cuando le echaba un pedo a alguien, decían: Tezcatlipoca echó un pedo.
# references = A&D/174:3
1	No	no	ADV	_	_	3	advmod	_	_
2	ipan	pan	NOUN	_	NounType=Relat|Number[psor]=Sing|Person[psor]=3|State=Pos	3	_	_	_
3	qujmatia	matia	VERB	_	Number[obj]=Sing|Number[subj]=Plur,Sing|Person[obj]=3|Person[subj]=3|Subcat=Tran|Tense=Pres|VerbForm=Fin	0	root	_	_
4	,	,	PUNCT	_	_	5	punct	_	_
5	ca	cah|ca	VERB	_	_	3	_	_	_
6	ixiptla	ixiptlatl	NOUN	_	Number[psor]=Sing|Person[psor]=3|State=Pos	5	_	_	_
7	in	in	DET|SCONJ	_	_	8	det	_	_
8	tlacateculotl	tlacatecolotl	NOUN	_	State=Abs	6	nmod	_	_
9	,	,	PUNCT	_	_	10	punct	_	_
10	tezcatlipuca	Tezcatlipoca	PROPN	_	_	8	appos	_	_
11	:	:	PUNCT	_	_	20	punct	_	_
12	auh	auh	CCONJ	_	_	20	cc	_	_
13	in	in	SCONJ	_	_	15	_	_	_
14	jquac	ihcuac	ADV	_	_	15	_	_	_
15	mjexi	iyexi	VERB	_	Number[subj]=Plur,Sing|Person[subj]=3|Reflex[obj]=Yes|Subcat=Tran|Tense=Pres|VerbForm=Fin	20	advcl	_	_
16	,	,	PUNCT	_	_	18	punct	_	_
17	in	in	DET|SCONJ	_	_	18	_	_	_
18	teiexi	iyexi	VERB	_	Animacy[obj]=Hum|Number[subj]=Plur,Sing|Person[obj]=3|Person[subj]=3|Subcat=Tran|Tense=Pres|VerbForm=Fin	15	_	_	_
19	,	,	PUNCT	_	_	20	punct	_	_
20	qujtoaia	ihtoa	VERB	_	Aspect=Imp|Number[obj]=Sing|Number[subj]=Plur,Sing|Person[obj]=3|Person[subj]=3|Subcat=Tran|Tense=Past|VerbForm=Fin	3	conj	_	_
21	:	:	PUNCT	_	_	22	punct	_	_
22	omjex	iyexi	VERB	_	Aspect=Prior|Number[subj]=Plur,Sing|Person[subj]=3|Reflex[obj]=Yes|Subcat=Tran|Tense=Past|VerbForm=Fin	20	ccomp	_	_
23	in	in	DET	_	_	24	det	_	_
24	tezcatlipuca	Tezcatlipoca	PROPN	_	_	22	nsubj	_	_
25	.	.	PUNCT	_	_	3	punct	_	_
```

**Comment by ftyers** [comment]
López Austin:
> También de él sabían que era la imagen del dios Tezcatlipoca. Y cuando se peía, cuando peía a alguno, decían: "Se peyó Tezca­tlipoca."

**Comment by ftyers** [comment]
Questions and comments:
- `ccomp` should be `parataxis:speech`
- Is *ca* `PART` or `AUX` ?
- What is the relation between *miyexi* and *teyexi*, is it `parataxis:shared`?

**Comment by ftyers** [comment]
Update:
![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/f6d711a6-1f46-4bf6-ba63-752797d64491)

Notes: 
- `conj` for *miyexi* and *tiyexi* because they both are `advcl` to *quihtoaya*
- `PART` for *ca*, because *ca* isn't used as a copula like that(?)

I'm still not 100% about *quimatia, ca in ixiptla in tlacatecolotl* part, but the rest is ok I think...

**Comment by Lguyogiro** [comment]
LGTM. I think the quimatiya ca ixiptla part makes sense.

**Comment by ftyers** [comment]
Merged in 46e3f27.
